File: 2025-1/sesion-de-clase-07/SoftProg/SoftProgRRHHModel/src/main/java/pe/edu/pucp/inf30/softprog/rrhh/model/Empleado.java
package pe.edu.pucp.inf30.softprog.rrhh.model;

/**
 *
 * @author eric
 */
public class Empleado extends Persona {
    private String cargo;
    private double sueldo;

    public String getCargo() {
        return cargo;
    }

    public void setCargo(String cargo) {
        this.cargo = cargo;
    }

    public double getSueldo() {
        return sueldo;
    }

    public void setSueldo(double sueldo) {
        this.sueldo = sueldo;
    }
}

File: 2025-2/sesion-de-clase-21/dotnet/SoftProgModelo/RRHH/Empleado.cs
namespace PUCP.SoftProg.Modelo.RRHH {
    public class Empleado : Persona {
        public Area Area { get; set; }
        public CuentaUsuario CuentaUsuario { get; set; }
        public Cargo Cargo { get; set; }
        public double Sueldo { get; set; }
    }
}
